src/stdatamodels/jwst/datamodels/tests/test_models.py

```diff
@@ -258,16 +258,17 @@ def test_slit_from_image():
     # assert not hasattr(slit_dm, 'meta')
 
     slit = SlitModel(im)
+    assert isinstance(slit, SlitModel)
     assert_allclose(im.data, slit.data)
     assert_allclose(im.err, slit.err)
     assert hasattr(slit, "wavelength")
     assert slit.meta.instrument.name == "MIRI"
 
     im = ImageModel(slit)
-    assert type(im) is ImageModel
+    assert isinstance(im, ImageModel)
```

Comment:
Did something require changing `type(im) is ImageModel` to this `isinstance(im, ImageModel)`. The former is more strict (doesn't allow `ImageModel` subclasses).

Reply:
no there was no reason, my IDE just didn't like it. will revert

Reply:
https://github.com/spacetelescope/stdatamodels/pull/511/commits/14d4073152489323a9f157ca39f669ea1d2d24a0